dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -252,7 +252,7 @@ protected void postProcessAfterScopeModelChanged(ScopeModel oldScopeModel, Scope
         }
         if (CollectionUtils.isNotEmpty(this.registries)) {
             this.registries.forEach(registryConfig -> {
-                if (registryConfig.getScopeModel() != applicationModel) {
+                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
```

Comment:
**Security Issue**: Null check added for registryConfig before accessing its methods could prevent potential NullPointerException. <br> **Fix**: Ensure that objects are checked for null before method access to prevent NullPointerException. <br> **Code Suggestion**: 
 ```
 -        return StringUtils.isEmpty(getGroup())
 -                ? (interfaceConfig != null ? interfaceConfig.getGroup() : getGroup())
 -                : getGroup();
 +        if (StringUtils.isEmpty(getGroup()) && interfaceConfig != null) {
 +            return interfaceConfig.getGroup();
 +        }
 +        return getGroup();
 ```